ml/backend/ggml/ggml/src/ggml-cpu/arch/s390/quants.c

```diff
@@ -0,0 +1,1057 @@
+#define GGML_COMMON_IMPL_C
```

Comment:
I'm not sure if many of these source files are built – do we need to include them?